plugins/audit_logs/girder_audit_logs/cleanup.py

```diff
@@ -0,0 +1,26 @@
+"""
+This script is for deleting old audit log entries from the database. Example invocation:
+
+    girder audit_logs_cleanup --days=30 --types=rest.request
+"""
+
+import click
+import datetime
+from girder_audit_logs import Record
+
+
+@click.command(name='audit_logs_cleanup')
```

Comment:
Should this and the docstring be changed as well? (for both files)

Reply:
Fixed